app/forms/quota_search_form.rb

```diff
@@ -33,7 +33,7 @@ def large_result?
   end
 
   def geographical_area
-    GeographicalArea.countries.find do |country|
+    GeographicalArea.all.find do |country|
```

Comment:
So we are not looking just in countries anymore?

Reply:
GeographicalArea.all does call the `/geographical_areas/countries` endpoint bizarrely